src/interfaces/threadMessage.ts

```diff
@@ -0,0 +1,8 @@
+import { APP_DATA_TYPES } from '@/config/appDataTypes';
+
+export interface ThreadMessage {
```

Comment:
Where do we use this, And why does it reference the `APP_DATA_TYPES` ?

Reply:
This interface is used in `ChatbotPrompts.tsx` and in `src/utils/chatbot.ts`. It is used to construct the prompt for ChatGPT with the initial prompt, and the history of messages. I updated the `type` from `APP_DATA_TYPES` to a more restrictive values `APP_DATA_TYPES.BOT_COMMENT | APP_DATA_TYPES.COMMENT`. 

The usage of APP_DATA_TYPES values allow the `buildPrompt` to construct the history with defining which message is the assistant and which is the user.